server/models/books.js

```diff
@@ -31,4 +31,55 @@ export default class Books {
     [title, body, description, genre, pages, author]);
     return rows[0];
   }
+
+  /**
+   * Store book request made by user to the database
+   *
+   * @static
+   * @async
+   * @name createBookRequest
+   * @param {object} bookRequestEntity - Details of the book request
+   * @returns {object} Details of the book request entity from database
+   * @memberof Books
+   */
+  static async createBookRequest(bookRequestEntity) {
+    const { userId, title, author } = bookRequestEntity;
+    const { rows } = await pool.query(
+      `INSERT INTO book_request
+        ("userId", title, author) 
+      VALUES
+        ($1, $2, $3)
+      RETURNING *`,
+      [userId, title.toLowerCase(), author]
+    );
+
+    return rows[0];
+  }
+
+  /**
+   * Store book request made by user to the database
+   *
+   * @static
+   * @async
+   * @name findABookRequest
+   * @param {string} bookTitle - Title of the book that was requested
+   * @returns {object} Details of the book that was searched
+   * @memberof Books
+   */
+  static async findABookRequest(bookTitle) {
+    const { rows } = await pool.query(
+      `SELECT
+        id,
+        "userId",
```

Comment:
Nice work @abdulfatai360 . Just for clarity, is there a reason why userId is in quotes and the other fields are not quoted?

Reply:
Thanks fr your feedback @chiboycalix . I put the qoutes because I want to maintain the camel case of the property. If I had removed it, Postgres would change that to "userid" - _in lowercase_. Also, if you look at the database config file, you would see I use the qoutes around "userid" to create the column on the book_request table. It's a trick to make Postgres maintain the camel case stuff

Reply:
wao, am just getting to know this. Thank you